mcp/src/apis/getUsers.ts

```diff
@@ -5,13 +5,22 @@ import { ApiFactory } from '../shared/boilerplate/src/types.js';
 const inputSchema = {
   includeTimezone: z
     .boolean()
+    .default(true)
```

Comment:
Applying a default makes it optional, which breaks things. See https://github.com/timescale/tiger-slack-mcp-server/pull/13